cmake/raylib-config.cmake

```diff
@@ -26,7 +26,7 @@ find_path(raylib_INCLUDE_DIR
 )
 
 find_library(raylib_LIBRARY
-    NAMES raylib
+    NAMES raylib raylib_static
```

Comment:
That way you lose the ability to influence whether you get the static or shared raylib on other platforms, no?
Can you make adding `raylib_static` to `NAMES` dependent on `raylib_USE_STATIC_LIBS`?

Reply:
@a3f Thanks for the review! I have update the PR.